### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by Feliverse**
**Learning objectives**

- Use React Router.
- Use React components.

**Project requirements**

- Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
- Create a directory for your reusable /components.
- Add [React Router](https://reactrouter.com/en/main/start/tutorial#setup)
- [Setup your router](https://reactrouter.com/en/main/start/tutorial#adding-a-router)
- Add two <Route>s to your application:
- /books - the default view:
- Should display a list of books
- Should display a form for adding a book (no functionality yet).
- /categories:
- Should display a button "Check status" only.
- Create a <Navbar> component that includes <Link>s to the two routes you created
- Create a <Book> component that displays a single book (props are title and author.
- Each Book has a "remove" button (no functionality yet)
- Create a <Form> component (with inputs for a title and an author [IMPORTANT in the design you can see an input for a category - please replace it with an author.]).

### CODE REVIEW

File: src/components/Categories.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+function Categories() {
```
Comment (topeogunleye) [inline src/components/Categories.js:1]: - [x] As required, I kindly suggest that you have a `<Button>` on your Categories page with the text "Check status"

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @Feliverse, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Kindly remove the `postgresql` and `express.js` under your Tech Stack.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Categories.js (hunk 1 above)
Comment (Feliverse) [reply src/components/Categories.js:1]: Hi @topeogunleye, 
Thank you

**Review by EricMbouwe — APPROVED** [review]
Hi @Feliverse, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Requested changes ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**